src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -31,12 +31,18 @@ const AdContainer = styled.section`
   ${({ slotType }) => (slotType === 'mpu' ? mpuStyles : leaderboardStyles)}
 `;
 
-export const getBootsrapSrc = queryString => {
+export const getBootstrapSrc = queryString => {
   const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
   const params = useLiveSrc ? '' : 'test/';
   return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap.js`;
 };
 
+export const getBootstrapLegacySrc = queryString => {
```

Comment:
I think we can consolidate this function with `getBootstrapSrc` so that we do not have almost duplicate unit tests.

We could write a function that returns `https://gn-web-assets.api.bbc.com/ngas/${params}` and when you declare the script below you can just append the file name to the string. 

Something like this:

```
        <script type="module" src={getBootstrapSrc(queryString) + "dotcom-bootstrap.js" } />
        <script nomodule="nomodule" src={getBootstrapSrc(queryString) + "dotcom-bootstrap-legacy.js" }} />

```

Reply:
Refactored it and the tests a bit! Just put all scripts on env variables